packages/matplotlib/meta.yaml

```diff
@@ -28,11 +28,11 @@ build:
     - png16=png
   post: |
     cd build/matplotlib-3.5.1/dist/matplotlib-3.5.1/
-    wget -O matplotlib/mpl-data/fonts/ttf/Humor-Sans.ttf http://antiyawn.com/uploads/Humor-Sans-1.0.ttf
```

Comment:
We were downloading this font every time from someone's personal website.
It is [not mandatory for matplotlib installation](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.xkcd.html), but for backward compatibility, I embedded this font in our repository.